src/js/ui_components/confirmation_modal.js

```diff
@@ -0,0 +1,112 @@
+import {Component} from "../utils";
+
+export class ConfirmationModal extends Component {
+    static  COOKIE_NAMES = {
+        BOUNDARY_TYPE_SELECTION: 'boundary_selection',
+        DATA_MAPPER_VERSION_SELECTION: 'version_selection'
```

Comment:
don't worry about this now, but yo don't want a generic confirmation modal to be aware of cookie names of each use case of it.

You'd rather want to instantiate it with a cookie name which is owned by the use case.